src/components/AppLayout/Header/index.tsx

```diff
@@ -67,7 +67,7 @@ const HeaderComponent = (): React.ReactElement => {
         connected={available}
         network={network}
         onDisconnect={onDisconnect}
-        openDashboard={openDashboard()}
+        openDashboard={!!openDashboard()}
```

Comment:
what is this prop supposed to do?

By the `!!` it looks like it should be a boolean value, but from the `openDashboard()` function return value, it seems to be returning either a `false` value or a `function` or an `undefined`.

Ah... if you see the `UserDetails` component, `openDashboard` is being used as a `function` if it's defined [`29f82ea` (#1449)](https://github.com/gnosis/safe-react/pull/1449/commits/29f82eac6878f229d79cfd8be7d8652cb544ec84#diff-a8baadfc3e39411c671e2433d2a7c73cR172-R176)

You can either change the `UserDetails` props type to `openDashboard?: boolean | () => void | null` and leave it as it was; or split this prop into to props, one for a flag and the other for the function.